modules/samples/firealarm/feature/feature/src/main/resources/agent/read-sensor.lua

```diff
@@ -0,0 +1,74 @@
+DHT= require("dht_lib")
```

Comment:
This is a lua script file which is going to upload into a device which has only 56kbits of memory. So I'm not seeing any point to add our license text to lua scripts which we going to upload to the device. Since we don't have any other option rather than adding full license header, I kept this file without having our standard license header.